app/components/layout/top-nav.tsx

```diff
@@ -127,21 +126,24 @@ export function LogoutButton({ className }: { className?: string }) {
 }
 
 export function TopNavUserMenu() {
+  const currentUser = useCurrentUser();
   return (
     <div>
       <StyledMenu
         button={
-          <div className="w-8">
-            <UserCircleIcon />
+          <div className="w-9 h-9 border-2 border-white rounded-full flex items-center justify-center font-extrabold">
```

Comment:
Might be good for a11y to add an aria-label here to say that the button is an "account button" for those using a screen reader.